learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,210 @@
+      "Toys & Games":1
+    },
+    "rating":{
+      "0":7,
+       …
+      "5":3
+    }
+  },
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+}
+```
+
+### Facet distribution
+
+The `facetDistribution` object contains the number of matching documents distributed among the values of a given facet. The `facets` search parameter expects an array of strings. Each string is an attribute present in the `filterableAttributes` list.
+
+The following response shows the facet distribution when searching for `led`:
+
+```json
+{
+  …
+ "facetDistribution":{
+    "brand":{
+      "AceLite":1,
+      "DALS Lighting":2,
+      "Globe Electric":1,
+      "Hyperikon":2,
+      "LUXRITE":1,
+      "Lithonia Lighting":1,
+      "MODOAO":2,
+      "Nadair":1,
+      "POCKETMAN":2,
+      "QPLUS":1,
+      "Sunlite":2,
+      "Suppromo":1,
+      "TORCHSTAR":1,
+      "Vinkki":1,
+      "WEWILL":1,
+      "hykolity":2,
+      "iHomma":1,
+      "iLintek":1
+    },
+    "category":{
+      "Home & Kitchen":1,
+      "Tools & Home Improvement":22,
+      "Toys & Games":1
+    },
+    "rating":{
+      "0":7,
+      "1":1,
+      "3.6":1,
+      "3.8":1,
+      "4":2,
+      "4.1":2,
+      "4.4":1,
+      "4.5":2,
+      "4.6":1,
+      "4.7":3,
+      "5":3
+    }
+  }
+  …
+}
+```
+
+`facetDistribution` contains an object for every given facet, in this case, `brand`, `category`, and `rating`. For each of these facets, there is another object containing all the different values and the count of matching documents. Note that zero values will not be returned: if there are no `led` products under the `Electronics` category, it is not displayed.
+
+::: note
+By default, `facets` returns a maximum of 100 facet values for each faceted field. You can change this value using the `maxValuesPerFacet` property of the [`faceting` index settings](/reference/api/settings.md#faceting).
+:::
+
+### Facet stats
+
+When using the `facets` parameter, any matching documents with facets containing numeric values are displayed in a `facetStats` object. `facetStats` contains the numeric minimum (`min`) and maximum (`max`) values per facet for all documents matching the search query. This can be used to create a range slider component allowing users to select a range of values for a facet.
+
+If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object.
+
+Meilisearch ignores string values like `"21"` when computing `facetStats`.
+
+The following response shows the `facetStats` when searching for `led`:
+
+```json
+{
+  …
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+  …
+}  
+```
+
+Since `rating` was the only numeric facet in our example, it is returned in the `facetStats` object.
+
+## Facet types
+
+### Single select facets
+
+Single-select or conjunctive facets use the `AND` logical operator. They only allow the selection of a single value per facet.
```

Comment:
This is confusing. I'm not sure this is the case, nor if this concept exists. It also implies you can only select one facet.

I don't think we should use `single-select` wording. Instead `conjunctive facet search`.